.github/workflows/check_export.yml

```diff
@@ -0,0 +1,36 @@
+name: check-export
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-requirements:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+      - run: |
+          for f in $(ls -1); do
+            if [[ -d $f ]]; then
+              if [[ ! -f $f/requirements.txt ]]; then
+                echo "$f/requirements.txt is MISSING"
+                exit 1
+              fi
+              python -m venv venv
+              source venv/bin/activate
+              # pin reflex version - no moving target
+              pip install reflex==0.2.7
+              pip install -r requirements.txt
+              reflex init
+              reflex export
+              deactivate
+              if ls frontend.zip backend.zip; then
```

Comment:
if possible, we should check that these are valid zips as well

Reply:
Sure doing `unzip -t` is easy enough.